src/hotspot/cpu/ppc/continuationFreezeThaw_ppc.inline.hpp

```diff
@@ -507,10 +507,8 @@ template<typename FKind> frame ThawBase::new_stack_frame(const frame& hf, frame&
     frame f(frame_sp, hf.pc(), frame_sp, fp);
     // we need to set the locals so that the caller of new_stack_frame() can call
     // ContinuationHelper::InterpretedFrame::frame_bottom
-    intptr_t offset = *hf.addr_at(ijava_idx(locals)) + padding;
-    assert((int)offset == hf.interpreter_frame_method()->max_locals() + frame::metadata_words_at_top + padding - 1, "");
-    // set relativized locals
-    *f.addr_at(ijava_idx(locals)) = offset;
+    // copy relativized locals from the heap frame
```

Comment:
Maybe reduce the comment?
```
    // we need to copy the locals so that the caller of new_stack_frame() can call
    // ContinuationHelper::InterpretedFrame::frame_bottom
```

Reply:
I think the comment is good as it is, because it describes just why we need to set the locals this early (and not the rest of the members in the fixed_frame).

Reply:
So the current version

```
    // we need to set the locals so that the caller of new_stack_frame() can call
    // ContinuationHelper::InterpretedFrame::frame_bottom
    // copy relativized locals from the heap frame
```

says that the locals are set and then again that they are copied. I thought that could be combined to what I suggested.
But I'm ok if you still prefer the current version.